libsolidity/interface/CompilerStack.cpp

```diff
@@ -1217,7 +1217,7 @@ Json CompilerStack::ethdebug(Contract const& _contract, bool _runtime) const
 	evmasm::LinkerObject const& object = _runtime ? _contract.runtimeObject : _contract.object;
```

Comment:
Are you sure that the problem isn't technically already here? And you should make sure not to ever even get here for interfaces?